### PR TITLE
seth: 0.8.2

### PR DESCRIPTION
**Pull request by asymmetric**
Release #214.

### CODE REVIEW

File: src/seth/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.2] - 2019-05-23
+### Fixed
+- Regression in `seth send` when `ETH_RPC_ACCOUNTS` was set
+
+## [0.8.1] - 2019-05-7
+### Fixed
+- `seth call`, `seth send` and `seth estimate` were broken due to an update in
+  [0.8.0], which has been reverted
```
Comment (MrChico) [inline src/seth/CHANGELOG.md]: duplicate

Comment (asymmetric) [reply src/seth/CHANGELOG.md]: Don’t understand

Comment (MrChico) [reply src/seth/CHANGELOG.md]: These four lines are already here

Comment (MrChico) [reply src/seth/CHANGELOG.md]: Check below

File: src/seth/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
...
 ## [0.8.1] - 2019-05-7
 ### Fixed
 - `seth call`, `seth send` and `seth estimate` were broken due to an update in
```
Comment (MrChico) [inline src/seth/CHANGELOG.md]: here! @asymmetric

File: src/seth/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.2] - 2019-05-23
+### Fixed
+- Regression in `seth send` when `ETH_RPC_ACCOUNTS` was set
+
+## [0.8.1] - 2019-05-7
```
Comment (MrChico) [inline src/seth/CHANGELOG.md]:
```suggestion
- ## [0.8.1] - 2019-05-7
```

**Pull request merged**